Neos.Flow/Classes/ResourceManagement/Target/FileSystemSymlinkTarget.php

```diff
@@ -183,4 +148,36 @@ protected function setOption($key, $value)
 
         return parent::setOption($key, $value);
     }
+
+    private function publish(string $targetPathAndFilename, string $sourcePathAndFilename): array
+    {
+        $exception = null;
+
+        if (!file_exists(dirname($targetPathAndFilename))) {
+            Files::createDirectoryRecursively(dirname($targetPathAndFilename));
+        }
+
+        try {
+            if (Files::is_link($targetPathAndFilename)) {
+                Files::unlink($targetPathAndFilename);
+            }
+
+            if ($this->relativeSymlinks) {
+                $result = Files::createRelativeSymlink($sourcePathAndFilename, $targetPathAndFilename);
+            } else {
+                $temporaryTargetPathAndFilename = $targetPathAndFilename . '.' . Algorithms::generateRandomString(13) . '.tmp';
+                symlink($sourcePathAndFilename, $temporaryTargetPathAndFilename);
+                $result = rename($temporaryTargetPathAndFilename, $targetPathAndFilename);
+            }
+        } catch (\Exception $exception) {
+            $result = false;
+        }
+
+        if (Files::is_link($targetPathAndFilename) && realpath($targetPathAndFilename) === realpath($sourcePathAndFilename)) {
```

Comment:
can there be a situation where both `realpath` return `false`? (since `false === false`)
i only know that `realpath` doesnt like stream wrapper and returns false then ...

Reply:
Oh, well. Technically, yes, that could happen. But I think we can ignore that here.

Reply:
👍